src/SavedSearches/SavedSearchesServiceProvider.php

```diff
@@ -52,15 +51,36 @@ function (Application $app) {
             }
         );
 
+        $app['udb3_saved_searches_repo'] = $app->share(
```

Comment:
Al suffixen best met 'sapi2', dus 'udb3_saved_searches_repo_sapi2'. Dan is het binnenkort ineens duidelijk welke voor sapi2 vs. sapi3 queries dient.